test/support/utils_sort.h

```diff
@@ -118,10 +118,9 @@ static std::atomic<std::int32_t> KeyCount;
 //! One more than highest index in array to be sorted.
 static std::uint32_t LastIndex;
 
-//! Keeping Equal() static and a friend of ParanoidKey class (C++, paragraphs 3.5/7.1.1)
```

Comment:
I checked these chapters of c++11 standard, which is referenced here, and I was not able to find anything particular about using static in the context of a friend function. I suppose `static` is not needed.

Meanwhile, static functions lead to
> utils_sort.h:230:1: warning: unused function 'Equal' [-Wunused-function]